pkg/scyllaclient/client.go

```diff
@@ -288,6 +288,19 @@ func (c *Client) IsNativeTransportEnabled(ctx context.Context, host string) (boo
 	return resp.Payload, nil
 }
 
+func (c *Client) SchemaAgreement(ctx context.Context) (bool, error) {
```

Comment:
nit: start with a verb like `HasSchemaAgreement` so the caller can better identify what it does and what the returned bool means